thoth/package_extract/image.py

```diff
@@ -525,6 +524,48 @@ def get_image_size(path: str) -> int:
     return total_size
 
 
+def _get_cuda_version(path: str) -> dict:
+    """Get the cuda version."""
+    res = dict()
+    """
+    Sometimes this file can be not accessbile, in those cases we can grant access
+    # st = os.stat(path + "/usr/local/cuda/version.txt")
+    # os.chmod(path + "/usr/local/cuda/version.txt", st.st_mode | stat.S_IEXEC)
+    """
+    with open(path + "/usr/local/cuda/version.txt", "r") as f:
+        output = f.read()
+        lines = output.split()
+        if len(lines) != 3:
+            _LOGGER.info(
+                "Unable to detect CUDA version from usr/local/cuda/version.txt output: %r",
+                output,
+            )
+        else:
+            res["/usr/local/cuda/version.txt"] = lines[-1]
+
+    """
+    It looks like there is no way of running a command on the extracted image.
+    However, if someone tries, this can be a good starting point.
+    result = run_command("{}/nvcc --version".format(path), raise_on_error=False)
```

Comment:
@fridex I wanted to do, but this won't work because `path` is just the folder where the image was extracted. So, I think we can proceed with only reading the file.

Reply:
You will need to check where `nvcc` binary is installed - might be in `/usr/bin`, `/usr/local/bin`, `/opt/...`. You are accessing`  /nvcc` (binary in the root) that will, most probably, not exist in the extracted container.